package.json

```diff
@@ -131,7 +131,7 @@
     "copy-to-clipboard": "^3.3.2",
     "easy-peasy": "^5.1.0",
     "file-saver": "^2.0.5",
-    "history": "5.3.0",
+    "history": "5",
```

Comment:
I checked the lock file, it still installs the 5.3.0

Reply:
As it's most actual one, "5" doesn't mean to stick to 5.0.0, it's just most current one in range of major version 5